src/Core/Datatypes/Legacy/Field/LatVolMesh.h

```diff
@@ -1301,24 +1301,20 @@ template <class Basis>
 Core::Geometry::BBox
 LatVolMesh<Basis>::get_bounding_box() const
 {
-  Core::Geometry::Point p0(min_i_,         min_j_,         min_k_);
-  Core::Geometry::Point p1(min_i_ + ni_-1, min_j_,         min_k_);
-  Core::Geometry::Point p2(min_i_ + ni_-1, min_j_ + nj_-1, min_k_);
-  Core::Geometry::Point p3(min_i_,         min_j_ + nj_-1, min_k_);
-  Core::Geometry::Point p4(min_i_,         min_j_,         min_k_ + nk_-1);
-  Core::Geometry::Point p5(min_i_ + ni_-1, min_j_,         min_k_ + nk_-1);
-  Core::Geometry::Point p6(min_i_ + ni_-1, min_j_ + nj_-1, min_k_ + nk_-1);
-  Core::Geometry::Point p7(min_i_,         min_j_ + nj_-1, min_k_ + nk_-1);
-
-  Core::Geometry::BBox result;
-  result.extend(transform_.project(p0))
-    .extend(transform_.project(p1))
-    .extend(transform_.project(p2))
-    .extend(transform_.project(p3))
-    .extend(transform_.project(p4))
-    .extend(transform_.project(p5))
-    .extend(transform_.project(p6))
-    .extend(transform_.project(p7));
+  std::vector<Core::Geometry::Point> p(8);
+  p[0] = Core::Geometry::Point(min_i_,         min_j_,         min_k_);
+  p[1] = Core::Geometry::Point(min_i_ + ni_-1, min_j_,         min_k_);
+  p[2] = Core::Geometry::Point(min_i_ + ni_-1, min_j_ + nj_-1, min_k_);
+  p[3] = Core::Geometry::Point(min_i_,         min_j_ + nj_-1, min_k_);
+  p[4] = Core::Geometry::Point(min_i_,         min_j_,         min_k_ + nk_-1);
+  p[5] = Core::Geometry::Point(min_i_ + ni_-1, min_j_,         min_k_ + nk_-1);
+  p[6] = Core::Geometry::Point(min_i_ + ni_-1, min_j_ + nj_-1, min_k_ + nk_-1);
+  p[7] = Core::Geometry::Point(min_i_,         min_j_ + nj_-1, min_k_ + nk_-1);
+
+  for (int i = 0; i < 8; ++i)
+    p[i] = transform_.project(p[i]);
```

Comment:
Not one but two raw `8`s? Think of a better way...

Hint:
No `8`s:
```
std::vector<Core::Geometry::Point> p = { {min_i_, min_j_, min_k_}, {.....}, };

std::for_each(p.begin(), p.end(), [this](Core::Geometry::Point& x) { x = transform_.project(x); });
```

Reply:
I have to agree with Dan, there is definitely a much better way of doing this and his hint is a great start.